release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -144,6 +144,26 @@ As a cluster administrator, you can configure `internalTrafficPolicy=Local` on a
 
 For more information, see link:https://kubernetes.io/docs/concepts/services-networking/service-traffic-policy[Service Internal Traffic Policy].
 
+[id="ocp-4-11-new-metallb-CRDs-logging-control"]
+==== Support for new MetalLB custom resource defintions (CRDs) and control over logging verbosity
+
+MetalLB CRDs have been updated with some new CRDs to support more complex configuration. 
+
+The following new CRDs have been added: 
+
+* IPAddressPools
+* L2Advertisement
+* BgpAdvertisement
+
+With these enhancements, users can use the Operator for more complex configurations, for example isolating nodes, or segmenting the network. In addition enhancements made to the FRRouting (FRR) logging component allows you to control the verbosity of the logs generated.
+
+[NOTE]
+====
+CRDs documented in 4.10 and the existing ways to configure MetalLB as described in link:https://docs.openshift.com/container-platform/4.10/networking/metallb/about-metallb.html[About MetalLB and the MetalLB Operator] are still supported. 
```

Comment:
I'd say "still supported but deprecated"

Reply:
Hi @fedepaol I added this clarification.